translations/ga/census_household_gb_nir_ga.po

```diff
@@ -2,4301 +2,6502 @@ msgid ""
+#. Answer option
+msgctxt "Do you have any of the following <em>other health conditions</em> which have lasted, or are expected to last, at least 12 months?"
 msgid "An intellectual or learning disability"
 msgstr "Míchumas intleachtúil nó foghlama"
 
-#. answer-id: disability-other-answer
-msgctxt "Answer for: Do you have any of the following <em>other health conditions</em> which have lasted, or are expected to last, at least 12 months?"
+#. Answer option
+msgctxt "Do you have any of the following <em>other health conditions</em> which have lasted, or are expected to last, at least 12 months?"
 msgid "A learning difficulty"
 msgstr "Deacracht foghlama"
 
-#. answer-id: disability-other-answer
-msgctxt "Answer for: Do you have any of the following <em>other health conditions</em> which have lasted, or are expected to last, at least 12 months?"
+#. Answer option
+msgctxt "Do you have any of the following <em>other health conditions</em> which have lasted, or are expected to last, at least 12 months?"
 msgid "Autism or Asperger syndrome"
 msgstr "Siondróm uathachais nó Asperger"
 
-#. answer-id: disability-other-answer
-msgctxt "Answer for: Do you have any of the following <em>other health conditions</em> which have lasted, or are expected to last, at least 12 months?"
+#. Answer option
+msgctxt "Do you have any of the following <em>other health conditions</em> which have lasted, or are expected to last, at least 12 months?"
 msgid "An emotional, psychological or mental health condition"
-msgstr "Fadhb mhothúchánach, shíceolaíoch nó mheabhair-shláinte"
+msgstr "Fadhb mhothúchánach, shíceolaíoch nó mheabhairshláinte"
 
-#. answer-id: disability-other-answer
-msgctxt "Answer for: Do you have any of the following <em>other health conditions</em> which have lasted, or are expected to last, at least 12 months?"
+#. Answer option
+msgctxt "Do you have any of the following <em>other health conditions</em> which have lasted, or are expected to last, at least 12 months?"
 msgid "Frequent periods of confusion or memory loss"
 msgstr "Tréimhsí minice de mhearbhall nó de dhíchuimhne"
 
-#. answer-id: disability-other-answer
-msgctxt "Answer for: Do you have any of the following <em>other health conditions</em> which have lasted, or are expected to last, at least 12 months?"
+#. Answer option
+msgctxt "Do you have any of the following <em>other health conditions</em> which have lasted, or are expected to last, at least 12 months?"
 msgid "Long-term pain or discomfort"
 msgstr "Pian nó míchompord fadtéarmach"
 
-#. answer-id: disability-other-answer
-msgctxt "Answer for: Do you have any of the following <em>other health conditions</em> which have lasted, or are expected to last, at least 12 months?"
+#. Answer option
+msgctxt "Do you have any of the following <em>other health conditions</em> which have lasted, or are expected to last, at least 12 months?"
 msgid "Other condition"
 msgstr "Fadhb eile"
 
-#. answer-id: disability-
```

Comment:
Extra spaces in `em` tags need to be removed.

Reply:
changed